### DISCUSSION TITLE
Add a custom REST LLM API template to Garak Config yaml file

### DISCUSSION THREAD

**Discussion by EricXQiu** [NVIDIA/garak#1040] [Q&A]
Garak supports REST APIs but I would like to know how to add the request template in the yaml config file.

For example, the request template can be as follows:
```
{
    "messages": [
        {"role":"system", "content": "You are a helpful assistant."},
        {''role": "user", "content": <actual_text>}
    ]
}
```

How to pass this message to garak config yaml.

**Comment by leondz** [comment]
Could you open this discussion as a feature request issue? Thank you!
